src/Pos/CompileConfig/Type.hs

```diff
@@ -55,4 +55,10 @@ data CompileConfig = CompileConfig
       -- ^ Block retrieval queue capacity
     , ccPropagationQueueSize          :: !Int
       -- ^ InvMsg propagation queue capacity
+
+----------------------------------------------------------------------------
+-- Hardware/system
+----------------------------------------------------------------------------
+    , ccMemPoolLimitRatio             :: !Word
+      -- ^ This ratio affects mempool size limit.
```

Comment:
Okay, I will copy-paste my comment into all places, but I am almost sure it will diverge sooner or later.